packages/core/src/generators/react/blocks.ts

```diff
@@ -106,12 +106,13 @@ const NODE_MAPPERS: {
   },
   Show(json, options, component) {
     const wrap = wrapInFragment(json);
+    const wrapElse = true; // TODO: make this configurable
```

Comment:
I think you can implement this by doing:

```suggestion
    const wrapElse = wrapInFragment(json.meta.else as any)
```

Reply:
This doesn't work for me